libs/model/src/community/CreateStakeTransaction.command.ts

```diff
@@ -0,0 +1,153 @@
+import type { Command } from '@hicommonwealth/core';
+import { commonProtocol, schemas } from '@hicommonwealth/core';
+import { Op } from 'sequelize';
+import Web3 from 'web3';
+import { models } from '../database';
+import { mustExist } from '../middleware/guards';
+import { StakeTransactionInstance } from '../models/stake_transaction';
+
+async function createStake(
+  web3: Web3,
+  transactionHash: string,
+  communityId: string,
```

Comment:
Seems like `communityId` is just passed so that we can return it. If this is needed in the command response we should just modify the `map` call below to merge `communityId` instead of passing it here and not using it.

Reply:
Could you clarify this a bit? If we modify the map call below, we will need to return all the values for StakeTransaction from this function to make the db create request in the map body. Is this understanding correct?

Reply:
I don't know what I was thinking. Didn't see we used `communityId` in model. This is fine.